concurrency/MpmcHashmap.h

```diff
@@ -90,6 +90,7 @@ class MpmcHashmap32
         tombstone
     };
 
+public:
```

Comment:
maybe remove the next public:

Reply:
this was a mistake. Made it public only to print the size of entry in another place but it supposed to be private